spk/dnscrypt-proxy/src/update-blocklist.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Add's randomness when running cron job (0-20 minutes)
+# Be nice to servers and don't update at exactly midnight or exactly on an hour
+[ -n "$RANDOM" ] && sleep $(($RANDOM %1200))
+
+cd /var/packages/dnscrypt-proxy/target/var/ || exit
+python generate-domains-blacklist.py > blacklist.txt.tmp && mv -f blacklist.txt.tmp blacklist.txt
```

Comment:
I would point explicit python location with /bin/python
Again please rename everything as "blocklist" for consistency.